calculatorFrame.java

```diff
@@ -582,65 +574,64 @@ else if(e.getSource() == clearAll){
 			duplicateOperatorFlag = false;
 			textArea.setText("");
 		}
-		
+
 		//Clear Last
-		else if(e.getSource() == clearLast){
+		else if(e.getSource() == clearLast) {
 			String textAreaLength = textArea.getText();
 			String lastChar = textArea.getText().substring(textArea.getText().length() - 1);
 			//System.out.println(lastChar);
 			System.out.println("Flag is = " + flag);
-			if (flag == false){
-				if (lastChar.equals("+")){
+			if (flag == false) {
+				if (lastChar.equals("+")) {
 					plusFlag = false;
 					duplicateOperatorFlag = false;
-					textArea.setText(textAreaLength.substring(0, textAreaLength.length()-1));
-				}else if(lastChar.equals("-")){
+					textArea.setText(textAreaLength.substring(0, textAreaLength.length() - 1));
```

Comment:
I feel like this long line could be wrapped in a private method that better describes what it’s doing.

Should this be called `clearLastDisplayedCharacter()`?